packages/pv-scripts/scripts/dev.js

```diff
@@ -38,15 +38,15 @@ prepareWebpackConfig("development")
     const devServer = new WebpackDevServer(compiler, devServerConfig);
 
     // Launch WebpackDevServer.
-    devServer.listen(devServerConfig.port, "0.0.0.0", err => {
+    devServer.listen(devServerConfig.port, devServerConfig.host, err => {
       if (err) {
         return console.log(err);
       }
       if (isInteractive) {
         clearConsole();
       }
 
-      console.log(chalk.cyan(`Starting the development server on port ${devServerConfig.port}...\n`));
+      console.log(chalk.cyan(`Starting the development server: http://${devServerConfig.host}:${devServerConfig.port}\n`));
```

Comment:
you could also add a check, whether devServers option [https](https://webpack.js.org/configuration/dev-server/#devserverhttps) is set

Reply:
good point, its added